### ISSUE TITLE
GHC 8 cannot build tests for 5.2

### ISSUE THREAD

**Issue by ocharles** [ekmett/bifunctors#44]
```
Preprocessing test suite 'bifunctors-spec' for bifunctors-5.2...
[1 of 2] Compiling BifunctorSpec    ( tests/BifunctorSpec.hs, dist/build/bifunctors-spec/bifunctors-spec-tmp/BifunctorSpec.dyn_o )

tests/BifunctorSpec.hs:1:1: error:
    Exception when trying to run compile-time code:
      src/Data/Bifunctor/TH.hs:(341,1)-(356,70): Non-exhaustive patterns in function makeBiFunForCon

    Code: deriveBifoldable ''StrangeGADT
```

I see there are closed issues about GHC 8, so if this has already been fixed feel free to close this issue and shout at me!

**Comment by RyanGlScott** [comment]
This should be fixed as of [`bifunctors-5.2.1`](http://hackage.haskell.org/package/bifunctors-5.2.1).

I don't believe Stackage is using `5.2.1` due to [other issues](https://github.com/ekmett/bifunctors/issues/42) which are fixed in the repo (but awaiting a new Hackage release).

**Comment by RyanGlScott** [comment]
I just uploaded [`bifunctors-5.3`](http://hackage.haskell.org/package/bifunctors-5.3), which should fix #42 and thus become part of Stackage soon. If this doesn't happen, please reopen the issue.
